docs/_posts/2024-07-18-Run-your-pandas-workloads-on-cluster.md

```diff
@@ -0,0 +1,152 @@
+---
+layout: post
+title: "Running Pandas on multi node cluster with Modin"
+categories: misc
+permalink: /gh_page_4.html
+author: arunjose696
+---
+
+## Introduction
+
+Handling large datasets efficiently is a common challenge in data science. Traditional tools like Pandas can struggle with scalability, often becoming slow and memory-intensive with large datasets. This is where Modin and Ray come into play. Modin is a powerful dataframe library designed to scale Pandas operations by spreading the workload across multiple cores or nodes. It supports various backends such as Dask, Ray, and MPI, with Ray being the default choice. Ray a distributed computing framework ensures smooth and efficient data processing.
```

Comment:
```suggestion
Handling large datasets efficiently is a common challenge in data science.
Traditional tools like pandas can struggle with scalability, often becoming slow and
memory-intensive with large datasets. This is where Modin on Ray comes into play.
Modin is a powerful dataframe library designed to scale pandas operations by spreading the workload
across multiple cores or nodes. It supports various execution engines such as Ray, Dask and MPI,
with Ray being the default choice. Ray a distributed computing framework ensures smooth and
efficient data processing.
```

Reply:
Let's use lower case for pandas. Also, please split long lines into multiple ones.